### PR TITLE
chore: aggregate our stylish contributors for the upcoming 2.0 release

### PR DESCRIPTION
**Pull request by vytas7**

### CODE REVIEW

**Comment by vytas7** [comment]
I have linked contributors to their GitHub pages, but we can of course revert to the plain listing style used in `1.4.0`.

I have left out two contributors that have deleted their GitHub accounts since:
- *Bertrand Lemasle*
- *Patrick Schneeweis*

Should we just include their name/surname in plain text?

File: CHANGES.rst [hunk 1]
```diff
@@ -114,6 +114,47 @@ New & Improved
 Fixed
 -----
 
+(None)
```
Comment (vytas7) [inline CHANGES.rst]: Note that this won't be `(None)` as we aggregate all the changes.

**Comment by kgriffs** [comment]
> Should we just include their name/surname in plain text?

Yes, let's just include their full names without a link. Everything else LGTM.

**Comment by vytas7** [comment]
Included the said contributors without a GH account.

**Pull request merged**